builtins/amp-img.md

```diff
@@ -153,7 +153,7 @@ If you want your image to display at a fixed size:
 2. Specify the `width` and `height`.
 
 {% call callout('Read on', type='read') %}
-Learn about the [inferred layout](https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn’t-specified?) if you don't specify the `layout` attribute.
+Learn about the [inferred layout](https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn%E2%80%99t-specified?) if you don't specify the `layout` attribute.
```

Comment:
I don't think it's necessary to use escaped text. From what I saw, this works with just a non-pretty apostrophe.

```
https://www.ampproject.org/docs/guides/responsive/control_layout#what-if-the-layout-attribute-isn't-specified?
```

Reply:
No it doesn't - if you use the simple apostrophe you get top of page not the anchor tag.

Reply:
Aha, in that case, I stand corrected. (I merely checked to make sure the link checker was satisfied.)

Reply:
%$#% unicode :-)

Reply:
I don't have write, so if could merge it that would be great.

Reply:
@jpettitt Looks like @bpaduch fixed this (among other things) in #13929, so this PR is now redundant.

Reply:
ok thx